haystack/nodes/document_classifier/transformers.py

```diff
@@ -94,27 +98,34 @@ def __init__(
         self.return_all_scores = return_all_scores
         self.labels = labels
         self.task = task
+        self.batch_size = batch_size
 
     def predict(self, documents: List[Document]) -> List[Document]:
         """
-        Returns documents containing classification result in meta field
+        Returns documents containing classification result in meta field.
+        Documents are updated in place.
 
         :param documents: List of Document to classify
         :return: List of Document enriched with meta information
         """
         texts = [doc.content for doc in documents]
```

Comment:
Why not picking here the texts based on `self.classification_field`? Then we can stick to `Document` objects and avoid the field map stuff from above

Reply:
done